currency-rates/internal/services/rate.go

```diff
@@ -7,6 +7,8 @@ import (
 	"go.uber.org/zap"
 )
 
+const ctxTime = 500 * time.Millisecond;
```

Comment:
I feel misunderstanding in naming.
It's really context time, but does it provide enough context to get what this constant is responsible for?

Will it be more clear with naming like apiCallTimeout or smth like that?

P.s This comment is non-blocking.